### PR TITLE
Resolved profile pic issue #167

### PR DESCRIPTION
**Pull request by kritikaparmar-programmer**
Fixes: #167 

# Proposed Changes
 - Changes in profile.html
 
# Additional Info
  - Now profile pic is adding
  
# Screenshots
 
profic

### CODE REVIEW

**Comment by kritikaparmar-programmer** [comment]
Hey @harsh-9in and @Kajol-Kumari I've done the changes Take a look now ..

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@kritikaparmar-programmer are you sure you have pushed the right change in this PR?  I am still facing the same issue when i tried pulling this code and runing it.

**Comment by kritikaparmar-programmer** [comment]
Hey @Kajol-Kumari take a look now Sorry for inconvenience.

**Comment by Kajol-Kumari** [comment]
@kritikaparmar-programmer I tested your PR and though the functionality is working fine, you have messed up the UI somehow.
Screenshot from 2020-09-25 22-32-19

only the `save change` functionality is not working as we are able to get the dialogue box for image change and so you just need to fix that, rest we don't need any other form as we alredy have one for the image update.

**Comment by kritikaparmar-programmer** [comment]
Hey @Kajol-Kumari take a look now. I removed the save changes button  as there was no option left.

File: mysite/users/templates/user/profile.html [hunk 1]
```diff
@@ -16,6 +16,27 @@
             Profile Image</span>
         </div>
 
+
```
Comment (Kajol-Kumari) [inline mysite/users/templates/user/profile.html:19]: @kritikaparmar-programmer why do we need this code snippet from here to [here](https://github.com/harsh-9in/Blogger/pull/175/files#diff-8208737725c422c84328b370f39b3b24R39) when we already have the the same code from [here](https://github.com/harsh-9in/Blogger/pull/175/files#diff-8208737725c422c84328b370f39b3b24R95). Also if you remove the `save` button how will the image get update in the backend and in the user profile?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]

**Comment by kritikaparmar-programmer** [comment]
Hey @Kajol-Kumari  and @harsh-9in Take a look now. I resolved the UI part and now the image is also updating.

**Review by Kajol-Kumari — APPROVED** [review]
Functionality is working fine now and it's good to go now :) 
Thanku @kritikaparmar-programmer for fixing this with patience.

**Review by harsh-9in — APPROVED** [review]

**Pull request merged**